restart_me.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pprint
+import tempfile
+import subprocess
+import contextlib
+
+import requests
+
+
+# https://stackoverflow.com/questions/6194499/pushd-through-os-system
+@contextlib.contextmanager
+def pushd(new_dir):
+    previous_dir = os.getcwd()
+    os.chdir(new_dir)
+    try:
+        yield
+    finally:
+        os.chdir(previous_dir)
+
+
+with tempfile.TemporaryDirectory() as tmpdir, pushd(tmpdir):
+    subprocess.run(
+        ["git", "clone", "--depth=1", "https://github.com/regro/circle_worker.git"],
```

Comment:
Should we go directly to the raw source and see if it exists? I guess maybe that is less stable?

Reply:
I’ve found that the raw src can be out of date after a push. The repo itself is always up to date. Plus the pull here is small so it should be fast.